### PR TITLE
Fix #3261: use the state the host is about to be in, not the one it's already in for power action notifications

### PR DESCRIPTION
**Pull request by skottler**

### CODE REVIEW

File: app/controllers/hosts_controller.rb [hunk 1]
```diff
@@ -211,7 +211,7 @@ def cancelBuild
   def power
     return invalid_request unless PowerManager::SUPPORTED_ACTIONS.include?(params[:power_action])
     @host.power.send(params[:power_action].to_sym)
-    process_success :success_redirect => :back, :success_msg => _("%{host} is now %{state}") % { :host => @host, :state => _(@host.power.state) }
+    process_success :success_redirect => :back, :success_msg => _("%{host} is about to enter the %{state} state") % { :host => @host, :state => _(params[:power_action].downcase) }
```
Comment (domcleal) [inline app/controllers/hosts_controller.rb]: It's no longer the state, but the action, so is this accurate?

e.g. %{host} is about to %{action}

Comment (skottler) [inline app/controllers/hosts_controller.rb]: I originally uses that phrase, but it's awkward in some cases with the present participle. Now that I look at it again I think more commands make sense than those that don't so I'll change it.

**Comment by skottler** [comment]
@domcleal I've updated the language :bowtie:

File: app/controllers/hosts_controller.rb [hunk 2]
```diff
@@ -211,7 +211,7 @@ def cancelBuild
   def power
     return invalid_request unless PowerManager::SUPPORTED_ACTIONS.include?(params[:power_action])
     @host.power.send(params[:power_action].to_sym)
-    process_success :success_redirect => :back, :success_msg => _("%{host} is now %{state}") % { :host => @host, :state => _(@host.power.state) }
+    process_success :success_redirect => :back, :success_msg => _("%{host} is about %{action}") % { :host => @host, :action => _(params[:power_action].downcase) }
```
Comment (ohadlevy) [inline app/controllers/hosts_controller.rb]: arent we missing a 'to'?

host is about <to> start | reboot | shutdown | poweron ..?

Comment (skottler) [inline app/controllers/hosts_controller.rb]: Fixed.

**Comment by domcleal** [comment]
Merged as e07a21c9, thanks @skottler!

**Comment by domcleal** [comment]
Note, commit messages should use "fixes #9999" rather than "fix" to link to redmine.

**Comment by skottler** [comment]
@domcleal middle school me cringes at mixing tenses, but I'll suck it and use "fixes" next time :+1:

**Pull request closed without merging**